glue/core/component.py

```diff
@@ -536,7 +536,7 @@ def ndim(self):
         return len(self._data.shape)
 
     def __getitem__(self, key):
-        return self._data[key].compute()
+        return np.array(self._data[key].compute())
```

Comment:
I think it would be better to use ``np.asarray`` in case the result of the computation is already a Numpy array, otherwise it will be unnecessarily copied in this case.